drivers/nrf9160_gps/nrf9160_gps.c

```diff
@@ -158,6 +169,24 @@ static void gps_thread(int dev_ptr)
 
 		switch (raw_gps_data.data_id) {
 		case NRF_GNSS_PVT_DATA_ID:
+			if (gps_is_blocked(&raw_gps_data.pvt)) {
+				if (operation_blocked) {
+					/* Avoid spamming the logs. */
+					continue;
+				}
+
+				LOG_INF("Waiting for time window to operate");
+				LOG_DBG("PSM or eDRX needs to be enabled for "
```

Comment:
Don't split log messages. Makes it impossible to search for.

Reply:
Yep, fixed